File: C++/Iniciante/1005.cpp
#include <stdio.h>
#include <iostream>

int main() {
  
  double a, b, media, peso1 = 3.5, peso2 = 7.5, soma_pesos = peso1 + peso2;
  std::cin >> a;
  std::cin >> b;

  media = ((a * peso1) + (b * peso2)) / soma_pesos;

  printf("MEDIA = %.5lf\n", media);
  
}

File: C#/Iniciante/1005.cs
using System; 

class URI {

    static void Main(string[] args) { 

        double a = Convert.ToDouble(Console.ReadLine());
        double b = Convert.ToDouble(Console.ReadLine());

        double peso1 = 3.5;
        double peso2 = 7.5;
        double soma_pesos = peso1 + peso2;

        double media = ((a * peso1) + (b * peso2)) / soma_pesos;

        Console.WriteLine($"MEDIA = {media:F5}");

    }

}
